README.adoc

```diff
@@ -154,14 +154,14 @@ include::finish/systemproto/pom.xml[]
 
 The first few lines define the [hotspot=basicConfig file=0]`syntax`, [hotspot=basicConfig file=0]`package`, and [hotspot=basicConfig file=0]`option` basic configuration of the `.proto` file. The [hotspot=SystemService file=0]`SystemService` service contains the four service calls that will be implemented in the coming sections.
 
-The [hotspot=getProperty file=0]`getProperty` rpc defines the unary call. In this call, the client side sends a `SystemPropertyName` message to the server side that returns back a `SystemPropertyValue` message with the property value. The [hotspot=SystemPropertyName file=0]`SystemPropertyName` and [hotspot=SystemPropertyValue file=0]`SystemPropertyValue` message types define that the `propertyName` and `propertyValue` must be string.
+The [hotspot=getProperty file=0]`getProperty` rpc defines the unary call. In this call, the client side sends a `SystemPropertyPrefix` message to the server side that returns back a `SystemPropertyValue` message with the property value. The [hotspot=SystemPropertyPrefix file=0]`SystemPropertyPrefix` and [hotspot=SystemPropertyValue file=0]`SystemPropertyValue` message types define that the `propertyPrefix` and `propertyValue` must be string.
 
-The [hotspot=getPropertiesServer file=0]`getPropertiesServer` rpc defines the server streaming call. The client side sends a `SystemPropertyName` message to the server side. The server returns back a stream of `SystemProperty` messages. Each [hotspot=SystemProperty file=0]`SystemProperty` message contains a `propertyName` and a `propertyValue` strings.
+The [hotspot=getPropertiesServer file=0]`getPropertiesServer` rpc defines the server streaming call. The client side sends a `SystemPropertyPrefix` message to the server side. The server returns back a stream of `SystemProperty` messages. Each [hotspot=SystemProperty file=0]`SystemProperty` message contains a `propertyPrefix` and a `propertyValue` strings.
```

Comment:
A single value is not associated with a prefix but a single property (which has a name)